template.yaml

```diff
@@ -89,20 +79,15 @@ Resources:
     Type: AWS::Serverless::Function # More info about Function Resource: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-function.html
```

Comment:
Why do we have `OldAnswerNonCommittalCheckFunction` and `NewAnswerNonCommittalCheckFunction` when the CodeUri is same ?

Reply:
These are technically 2 different checks, capable of being called independently. Since they are basically running the same check but on different answers, I have kept them together as they both require the same resources(prompts/utils). In the future, we are planning to change the check output schema to support returning multiple values in the form of a json object.
For example, right now the check would be
```
"old_answer_non_committal": true,
```
Whereas in the future it would be something like
```
"non_committal": {
    "old_answer": true,
    "new_answer": false,
}
```
These 2 checkers would be eventually be replaced by one that does both.

Reply:
Okay well if they both will be unified and this current state is just temporary then it doesn't matter.